src/daemons/MetaDaemon.cpp

```diff
@@ -16,6 +16,7 @@
 #include "kvstore/NebulaStore.h"
 #include "meta/ActiveHostsMan.h"
 
+using nebula::operator<<;
```

Comment:
Great!  That's a good idea!

Reply:
I must add some explanation here.

The reason why we need to `using` this stream operator(function) explicitly, is that `HostAddr` is an alias type to `std::pair`, whose actual type is declared in namespace `std` but not namespace `nebula`. In other words, if `HostAddr` is an actual user defined type, instead of an alias, `using` is not necessary here, since the ADL rule(Argument Dependent Lookup) should have covered this.